### PR TITLE
Idea page features

### PR DESCRIPTION
**Pull request by naleksi**
Initial version to show Idea.state at the top of the page. Uses graphics instead of original <li> blocks. The graphics need to be replaced with pristine version with correct width.

### CODE REVIEW

File: app/views/ideas/show.html.haml [hunk 1]
```diff
@@ -1,10 +1,40 @@
...
+      });
+  });
 .grid_24
-  .state
-    %ul.horizontal
-      %li.active IDEA
-      %li LAKILUONNOS
-      %li LAKIESITYS
-      %li LAKI
+  - eteneminen_jpg = {"idea" => "eteneminen_1_idea.jpg", "lakiluonnos"=> "eteneminen_2_lakiluonnos.jpg", "lakiesitys"=> "eteneminen_3_lakiesitys.jpg", "laki"=> "eteneminen_4_laki.jpg"}[@idea.state]
```
Comment (arttu) [inline app/views/ideas/show.html.haml:36]: Personally, I would want to see this refactored to the Idea model at some point.
Idea.at[:idea][:image], Idea.at[:lakiluonnos][:image] or something...

Comment (naleksi) [inline app/views/ideas/show.html.haml:36]: I agree completely that this is hardly optimal, or beautiful (or even endurable :). However, I fail to see how it's model's business to know which jpg relates to which idea.state. IMO, it is always View's business. Only reason I can come up with why to pick it up from Model would be the need to use dynamic URLs, or URLs that are based on models state. Now they are not.

File: app/views/ideas/show.html.haml [hunk 2]
```diff
@@ -20,13 +50,16 @@
       = link_to I18n.t("ideas.links.edit_idea"), edit_idea_path(@idea)
   .grid_7.prefix_1.omega
     .voting.sans_serif
-      %h3 Tehdäänkö tästä laki?
-      = link_to "Kyllä", "#", class: "btn jaa-btn"
-      = link_to "Ei", "#", class: "btn ei-btn"
+      - unless @vote
+        %h3 Tehdäänkö tästä laki?
+        = link_to "Kyllä", "/vote/vote_for/#{@idea.id}", class: "btn jaa-btn"
```
Comment (naleksi) [inline app/views/ideas/show.html.haml:55]: Definitely should replaced with routes. I tried to setup some, but failed to produce good. This is free range for refactoring!

File: app/views/ideas/show.html.haml (hunk 1 above)
Comment (ljuti) [inline app/views/ideas/show.html.haml:36]: Sounds like there's a view helper to be extracted. Let's look into this when we meet, Aleksi.

**Pull request merged**